src/Tablebot/Plugins/Roll/Dice.hs

```diff
@@ -9,39 +9,59 @@
 -- This plugin contains the neccessary parsers and stucture to get the AST for an
 -- expression that contains dice, as well as evaluate that expression.
 --
--- The behind the scenes for the dice is split into four files.
+-- The behind the scenes for the dice is split into six files, two of which
+-- are for generating dice statistics.
 -- - DiceData - the data structures for the AST for dice
 -- - DiceFunctions - functionality for dealing with functions and processing
 --    them
 -- - DiceParsing - parsers for getting all the DiceData items
 -- - DiceEval - methods for evaluating elements from DiceData
+-- - DiceStats - filling the type classes and function needed to generate
+--    statistics on dice
+-- - DiceStatsBase - functions to process completed dice ranges
 --
 -- Below is the regex representing the parsing for the expressions, and
```

Comment:
Not something you need to change, but for future reference - it's usually safer to put these kinds of grammars in Backus-Naur form rather than regex due to the sheer number of regex standards and possible overloading of `?`.